.github/workflows/ci.yml

```diff
@@ -334,3 +332,20 @@ jobs:
       - name: Push load test result
         if:  ${{ success() && github.ref_name == 'v3'}}
         run: git push 'https://github-actions:${{ secrets.GITHUB_TOKEN }}@github.com/nginx/agent.git' benchmark-results:benchmark-results
+  publish-packages:
+    name: Publish NGINX Agent v3 packages
+    if: ${{ github.event_name =='pull_request' &&
+            github.base_ref == 'v3' &&
+            github.event.pull_request.merged == true &&
+            !github.event.pull_request.head.repo.fork }}
+    needs: [ lint, unit-test, performance-tests,
+             load-tests, official-oss-image-integration-tests,
+             official-plus-image-integration-tests,
+             race-condition-test ]
+    uses: ./.github/workflows/release-branch.yml
+    with:
+      packageVersion: "3.0.0"
+      packageBuildNo: "${{ github.run_number }}"
```

Comment:
Could we add the sha id to the end of the build number? I would make it easier to track what packages match to what commit. So something like `<run number>.<short sha id>`

Reply:
Updated, this will now have the value like: <run_number>-<commit_id>